WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/WNPRC_PurchasingModule.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2020 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.labkey.wnprc_purchasing;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.labkey.api.data.Container;
+import org.labkey.api.data.ContainerManager;
+import org.labkey.api.module.DefaultModule;
+import org.labkey.api.module.ModuleContext;
+import org.labkey.api.view.WebPartFactory;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+
+public class WNPRC_PurchasingModule extends DefaultModule
+{
+    public static final String NAME = "WNPRC_Purchasing";
+
+    @Override
+    public String getName()
+    {
+        return NAME;
+    }
+
+    @Override
+    public @Nullable Double getSchemaVersion()
+    {
+        return 20.000;
+    }
+
+    @Override
+    public boolean hasScripts()
+    {
+        return true;
```

Comment:
We don't have scripts yet for this module, correct? Let's return false until they're needed, and also return null from getSchemaVersion()

Reply:
done